ova/generate_ova.sh

```diff
@@ -167,6 +177,9 @@ function main() {
       local OVA_VERSION="${WAZUH_VERSION}_${ELK_VERSION}"
       echo "Version to build: ${WAZUH_VERSION}-${ELK_VERSION} with ${STATUS} repository."
       build_ova ${WAZUH_VERSION} ${OVA_VERSION}
+      if [[ "${cheksum}" = "yes" ]]; then
```

Comment:
Two things:
- Typo => `cheksum`.
- If you use a single =, you need to write the check as if [ "A" = "B" ] to avoid errors.

Reply:
@DFolchA

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/192/commits/5a72389d9ddbedad13d02074112ad463a9d9e133